include/libtorrent/bencode.hpp

```diff
@@ -88,6 +88,28 @@ namespace libtorrent
 
 	namespace detail
 	{
+		template <typename T, typename Cond = typename std::enable_if<
```

Comment:
yes, but I didn't know you want it inline, don't you think t's very big? (in case no -O3 is used)